lib/cucumber/rails/world.rb

```diff
@@ -7,7 +7,9 @@
 module Cucumber #:nodoc:
   module Rails #:nodoc:
     class World < ActionDispatch::IntegrationTest #:nodoc:
-      include Rack::Test::Methods
+      if Cucumber::Rails.config.use_rack_test_helpers
```

Comment:
If you'd like to, you could remove two lines by using `include Rack::Test::Methods if Cucumber::Rails.config.use_rack_test_helpers`. :)

Reply:
Yes, you're right. I will do this.